package.json

```diff
@@ -21,6 +21,7 @@
     "babel-eslint": "^5.0.0",
     "babel-plugin-check-es2015-constants": "^6.5.0",
     "babel-plugin-syntax-trailing-function-commas": "^6.5.0",
+    "babel-plugin-syntax-object-rest-spread": "^6.3.13",
```

Comment:
Thanks for the heads up. I should pay more attention instead of just throwing up a PR and moving on to the next thing. Updated and passing now.